js/script.js

```diff
@@ -99,13 +169,65 @@ document.querySelector('#book-form').addEventListener('submit', (e) => {
   const title = document.querySelector('#title').value;
   const author = document.querySelector('#author').value;
   const isbn = document.querySelector('#isbn').value;
+  const email = document.querySelector('#authorEmail').value;
+  const phone = document.querySelector('#author-phone').value;
 
   // Validate
-  if(title === '' || author === '' || isbn === '') {
-    UI.showAlert('Please fill in all fields', 'danger');
+  if (title === '') {
+    UI.show_error_text('.titleError');
+  }
+  if (author === '') {
+    UI.show_error_text('.authorError');
+  }
+  if (isbn === '') {
+    UI.show_error_text('.isbnError');
+  }
+  if (email === '') {
+    UI.show_error_text('.emailError');
+  }
+  if (phone === '') {
+    UI.show_error_text('.phoneError');
+  }
+
+  if(title === '' || author === '' || isbn === '' || email === '') {
```

Comment:
We are missing validation for phone.

<img width="811" alt="image" src="https://github.com/tamuir/book-list-js/assets/1679696/adee8eb8-930e-4d3d-889f-8f0f8c62ee38">

Reply:
The validations are not checking field types like number, email or numbers, if this is not optional please check & implement.

Also we can improve this and check multiple things (null, undefined, 0) as title === '' only checking for empty string, ex: 

if (!title || !author || !isbn || !email || !phone) {
  return;
}